test/lean/run_tests.py

```diff
@@ -46,6 +46,11 @@
     'config_abstract_bool', # Register type unsupported in state.ml
     'newtype',
     'assign_in_funarg',
+    'encdec', # Deactivated to enable match_bv
+    'encdec_subrange', # Deactivated to enable match_bv
+    'option_nest', # Deactivated to enable match_bv
+    'special_annot', # Deactivated to enable match_bv
+    'vector_subrange_pattern' # Deactivated to enable match_bv
```

Comment:
@ineol, these do not need to be disabled, no? `match_bv` is disabled by default.